miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/MiniAppMessageBridge.kt

```diff
@@ -583,6 +597,21 @@ open class MiniAppMessageBridge {
         bridgeExecutor.postError(callbackId, ErrorBridgeMessage.ERR_CLOSE_ALERT)
     }
 
+    private fun onDidReceiveMAAnalytics(callbackId: String, jsonStr: String) {
+        val maAnalyticsCallbackObj = Gson().fromJson(jsonStr, MaAnalyticsCallbackObj::class.java)
+        val successCallback = { message: String ->
+            bridgeExecutor.postValue(callbackId, message)
+        }
+        val errorCallback = { message: String ->
+            bridgeExecutor.postError(callbackId, message)
+        }
+        didReceiveMAAnalytics(
+            maAnalyticsCallbackObj.param.analyticsInfo,
+            successCallback,
+            errorCallback
+        )
```

Comment:
```suggestion
if (callbackObj?.param.analyticsInfo != null) {
        didReceiveMAAnalytics(
            maAnalyticsCallbackObj.param.analyticsInfo,
            successCallback,
            errorCallback
        )
        return
}
bridgeExecutor.postError(callbackId, "Invalid Format")
```

Reply:
@rleojoseph oh yeah, instead of checking only one case, i followed the format we used to follow, using try and catch. thanks for the suggestions though. addressed the issue here.